content/managing-apps.md

```diff
@@ -0,0 +1,346 @@
+---
+title: Installing, Updating, and Removing Applications
+description: >
+  Quick guide to installing, updating, and removing apps on your device.
+keywords:
+  - Apps
+  - Install
+  - Update
+  - Remove
+  - Delete
+
+facebookImage: /_social/article
+twitterImage: /_social/article
+
+hidden: false
+section: pop
+tableOfContents: true
+---
+
+## Installing & Uninstalling Applications via Terminal
+
+Launch the Terminal `SUPER` + `T` to Install applications using a command line interface. Terminal commands instruct package managers to retrieve application installation packages from repositories.
+
+### Flatpaks & .Deb Files
+
+**Flatpaks** will install a “containerized” version of the software. This means the software runs in its own sandbox, and the installation will include all dependencies and libraries required by the application. Flatpaks pull all dependencies from flathub.org, while installing with apt will pull from Ubuntu repos.
+
+Substitute `[packagename]` with the desired application (do not include brackets in the command).
+
+Apt install:
+
+```bash
+apt install [packagename]
+```
+
+Apt uninstall:
+
+```bash
+sudo apt remove [packagename]
```

Comment:
I'd recommend making this `autoremove` (or listing that separately) since that will also remove dependencies that are no longer needed.